app/views/faq_groups/basics.md

```diff
@@ -1,37 +1,36 @@
+The projects are open and transparent, and anyone can help build them, whether you want to contribute a few hours or a few months. Product ideas, vision and execution are driven by the community.
 
-Each contribution you make will be rewarded with a fair amount (as determined by the community) of stake in the project -- stake is distributed through App Coins. If a product is generating revenue, everyone who helped build it receives payment each month. During development, Assembly often helps cover expenses like servers and web domains.
+Each contribution you make will be rewarded with a fair amount (as determined by the community) of stake in the project. Stake is distributed in the form of App Coins. If a product is generating revenue, everyone who helped build it receives payment each month. During development, Assembly often helps cover expenses like servers and web domains.
 
 Working on Assembly projects is a great way to learn, challenge yourself and make a meaningful contribution to a great product. Also, if the products you help build are successful, you can earn substantial income.
 
 
 # What is the best way to get the feel for Assembly?
 
-Hop into a discussion and introduce yourself! There are lots of products in active development today([find some here!](https://assembly.com/discover)), and everyone working on them is friendly. Talk about your interests and availability, and see if there are any bounties you’d like to work on. Helping build products is a rewarding, fun way to quickly gain an understanding of how Assembly works.
+Hop into a discussion and introduce yourself! There are lots of products in active development today ([find some here!](https://assembly.com/discover)), and everyone working on them is friendly. Talk about your interests and availability and see if there are any bounties you’d like to work on. Helping build products is a rewarding, fun way to quickly gain an understanding of how Assembly works.
 
 Or submit your own idea [here](https://assembly.com/start)!
 
 
 # What parts of building can I help out with?
 
-Anything and everything. Take a look at the bounties on a particular project and see if anything is of interest for you. Be sure to say Hi in [general chat](https://assembly.com/chat/general) so that the Assembly community can welcome you, or to just introduce yourself to the team working on a project you like. No matter what your skills are, there’s someone on Assembly who needs them.
+Anything and everything. Take a look at the bounties on a particular project and see if anything is of interest to you. Be sure to say "hi" in [general chat](https://assembly.com/chat/general) so that the Assembly community can welcome you, or to just introduce yourself to the team working on a project you like. No matter what your skills are, there’s someone on Assembly who needs them.
```

Comment:
Maybe “be sure to pop into [Community Chat]” would be better? The “hi” part feels a little awkward to me.